docs/FAQs.md

```diff
@@ -57,7 +57,7 @@ When an organization contains a small amount of data, the backfilling process is
 
 - **Atlassian IP address ranges need whitelisting** - [Please see our IP allowlist documentation](https://github.com/atlassian/github-for-jira/blob/main/docs/ip-allowlist.md)
 
-- **GitHub Enterprise Server Edition does not currently support application access via SAML SSO** - Only Github Enterprise Cloud offers this ability. [Learn more about authentication with SAML single sign-on](https://docs.github.com/en/enterprise-cloud@latest/authentication/authenticating-with-saml-single-sign-on/about-authentication-with-saml-single-sign-on#about-oauth-apps-github-apps-and-saml-sso).
+- **GitHub Enterprise Server Edition does not currently support application access via SAML SSO** - Only Github Enterprise Cloud offers this ability. As a workaround, however, you can try login to your GitHub Enterprise Server Edition SSO first, then come back to the app and continue configuration as normal. [Learn more about authentication with SAML single sign-on](https://docs.github.com/en/enterprise-cloud@latest/authentication/authenticating-with-saml-single-sign-on/about-authentication-with-saml-single-sign-on#about-oauth-apps-github-apps-and-saml-sso).
```

Comment:
login to -> logging into

Reply:
> GitHub Enterprise Server Edition does not currently support application access via SAML SSO*

That's kinda misleading, it does support, but the user must be logged in prior.

> then come back to the app and continue configuration as normal

If the session is initialized, that won't help. They must reset their session.

Reply:
Are we planning to also update
https://support.atlassian.com/jira-cloud-administration/docs/connect-a-github-enterprise-server-account-to-jira-software/
https://support.atlassian.com/jira-cloud-administration/docs/github-enterprise-server-integration-faq/
?